tests/Microsoft.Identity.Test.Common/Core/Mocks/TokenCacheHelper.cs

```diff
@@ -15,22 +15,24 @@ internal class TokenCacheHelper
         public static long ValidExpiresIn = 28800;
         public static long ValidExtendedExpiresIn = 57600;
 
-        internal void PopulateCacheForClientCredential(ITokenCacheAccessor accessor)
+        internal void PopulateCacheForClientCredential(ITokenCacheAccessor accessor, int tokensQuantity = 1)
         {
-
-            var atItem = CreateAccessTokenItem();
-            accessor.SaveAccessToken(atItem);
+            for (int i = 1; i <= tokensQuantity; i++)
+            {
+                var atItem = CreateAccessTokenItem(string.Format(System.Globalization.CultureInfo.InvariantCulture, TestConstants.ScopeStrFormat, i));
+                accessor.SaveAccessToken(atItem);
+            }
         }
 
-        internal static MsalAccessTokenCacheItem CreateAccessTokenItem()
+        internal static MsalAccessTokenCacheItem CreateAccessTokenItem(string scopes = "")
```

Comment:
>"" [](start = 87, length = 2)

sting.Empty?

Reply:
Default parameter must be a compile-time constant. `string.Empty` is not.